docs/docusaurus/siteConfig.js

```diff
@@ -20,49 +20,55 @@ const users = [
     caption: 'User1',
     // You will need to prepend the image path with your baseUrl
     // if it is not '/', like: '/test-site/img/docusaurus.svg'.
-    image: '/img/docusaurus.svg',
-    infoLink: 'https://www.facebook.com',
+    image: '/img/magma-logo.svg',
+    infoLink: 'https://magmacore.org',
     pinned: true,
   },
 ];
 
 const siteConfig = {
-  title: 'Magma', // Title for your website.
+  title: 'Magma Documentation', // Title for your website.
+  disableTitleTagline: true, 
   tagline: 'Bring more people online by enabling operators with open, flexible, and extensible network solutions',
-  url: 'https://magma.github.io', // Your website URL
-  baseUrl: '/magma/', // Base URL for your project */
-  // For github.io type URLs, you would set the url and baseUrl like:
-  //   url: 'https://facebook.github.io',
-  //   baseUrl: '/test-site/',
 
   // Used for publishing and more
   projectName: 'magma',
-  organizationName: 'magma',
+  organizationName: 'magma',  
+  url: 'https://magmacore.org', // Your website URL
+  baseUrl: '/', // Base URL for your project */
```

Comment:
I'm worried about breaking existing links with these changes. Is the plan to keep publishing on gh-pages for now? Or should we coordinate a change to gh-pages to redirect

Reply:
We should probably coordinate a time to redirect.  Definitely do not want to break anything until we are prepared to move it over.  We're working on a CNAME for the docs to live at docs.magmacore.org.  I'm working with @jbryce on this one, so I'd say let's hold until then?

Reply:
Cool, just give me a ping when we're good to go on that front and I'll push the redirect to gh-pages.

Reply:
@xjtian Thanks again for reviewing.  I've fixed our DNS issues and I believe we're ready to land this.  Once we land it, we will be using docs.magmacore.org.

Reply:
One thing... once we make the switch, Magma will want to update github pages to redirect to docs.magmacore.org (https://docs.github.com/en/enterprise/2.13/user/articles/redirects-on-github-pages)

Reply:
Hi @xjtian - Just a ping on the above ^^ :)

Reply:
do we have CI set up to automatically deploy to docs.magmacore.org? I see 1.1.0 as the most recent available version on that site but our docusaurus is up to 1.3

Reply:
The demo version (https://competent-euler-513227.netlify.app/docs/basics/introduction.html) is pointing to the docs site that I forked on my own repo.  Thus the discrepancy.  I don't have access to approve netlify pointing to the magma repo. so we'll need @jbryce to work with me on that.